examples/agent-examples.yaml

```diff
@@ -0,0 +1,71 @@
+# Example: Agents and Agent Relationships
+
+'@context':
+  '@vocab': https://w3id.org/valueflows#
+  rdfs: http://www.w3.org/2000/01/rdf-schema#,
+  alice: https://alice.example/
+  bob: https://bob.example/
+  fablab: https://fablab.example/
+  coop: https://coop.example/
+
+'@graph':
+
+  - '@id': alice:e1721a61-cd47-4556-84b9-8b1b81da15bf
+    '@type': Person
+    name: 'Alice'
+    image: 'https://alice.example/avatar.png'
+    url: 'https://alice.example'
+    primaryLocation: 'https://somelocation.example'
+    note: 'Alice is a mechanical engineer who likes to work in the fablab.'
+
+  - '@id': bob:b90b0b77-09a2-42e2-8bd4-e9ae2c1c6172
+    '@type': Person
+    name: 'Bob'
+
+  - '@id': fablab:3129ca8b-fcda-45be-bbda-294dc924d3b9
+    '@type': Organization
+    name: 'Driftless Fablab'
+
+  - '@id': coop:3129ca8b-fcda-45be-bbda-294dc924d3b9
+    '@type': Organization
```

Comment:
since `vf` doesn't define Organization I think we should use `org:Person` to avoid confusions